src/arch/riscv/tlb.cc

```diff
@@ -262,6 +264,21 @@ TLB::createPagefault(Addr vaddr, Mode mode)
     return std::make_shared<AddressFault>(vaddr, code);
 }
 
+// TODO: merge createPagefault and createAddrfault
+Fault
+TLB::createAddrfault(Addr vaddr, Mode mode)
+{
+    ExceptionCode code;
+    if (mode == TLB::Read) {
+        code = ExceptionCode::LOAD_ACCESS;
+    } else if (mode == TLB::Write) {
+           code = ExceptionCode::STORE_ACCESS;
+    } else {
+            code = ExceptionCode::INST_ACCESS;
```

Comment:
```suggestion
        code = ExceptionCode::STORE_ACCESS;
    } else {
        code = ExceptionCode::INST_ACCESS;
```
Spacing may be wrong.

Reply:
done